packages/legacy-workbench/eslint.config.mjs

```diff
@@ -0,0 +1,71 @@
+import cypressPlugin from 'eslint-plugin-cypress';
+import googleConfig from 'eslint-config-google';
+
+const filteredGoogleRules = Object.fromEntries(
+    Object.entries(googleConfig.rules).filter(([key]) => key !== 'valid-jsdoc')
+);
+
+export default [
+    {
+        files: ['**/*.js'],
+        ignores: ['dist/**', 'node_modules/**'],
```

Comment:
Can you also add ignores to the libraries js files which are copied in the legacy.